### PR TITLE
[Functions] New DB/ORM fields

### PR DESCRIPTION
**Pull request by bolekk**
New set of fields to support additional parameters of onchain requests. See FUN-370 for details.

### CODE REVIEW

**Review by KuphJr — DISMISSED** [review]
LGTM but maybe @pinebit should also review just because of the DB migration?

**Review by justinkaseman — DISMISSED** [review]
Nice! Just ack'ing these changes

**Comment by bolekk** [comment]
Thanks I'll let @pinebit take a look too before merging.

File: core/store/migrate/migrations/0154_ocr2dr_requests_table.sql
Comment (pinebit) [inline core/store/migrate/migrations/0154_ocr2dr_requests_table.sql:3]: The filename does not match.

**Review by pinebit — APPROVED** [review]

**Review by pinebit — APPROVED** [review]

**Pull request merged**
